test/OpenTelemetry.Instrumentation.Grpc.Tests/OpenTelemetry.Instrumentation.Grpc.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Unit test project for OpenTelemetry Grpc for .NET instrumentation</Description>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net461</TargetFrameworks>
```

Comment:
.NET Core 2.1 is near end of life but it's not there yet. We could probably add a `netcoreapp2.1` target here and it might just work.

Even if .NET Framework support is not quite ready yet, there's still a benefit to supporting netstandard2.0 for older .NET Core applications.